shared/lib_battery.h

```diff
@@ -314,9 +314,6 @@ class battery_t {
     // Returns the % replacement if on a capacity schedule. Returns 0 for "none" or "calendar"
     double getReplacementPercent();
 
-    // Change the timestep of the battery and its component models
-    void ChangeTimestep(double dt_hr);
-
```

Comment:
So this might be a merge conflict, as this shouldn't be getting deleted. I'm noticing some other changes spread throughout such as `dt_hr` being changed back to `dt_hour` in battery state structs. And in `cmod_stateful` as well.

Reply:
Not sure how I managed to do that but thanks for the catch. Fixed those potential merge conflicts and pushed again.